src/main/cpp/io/util/Usb.cpp

```diff
@@ -56,6 +48,15 @@ void Usb::Open()
         }
     }
 
+    ret = libusb_set_configuration(m_dev, m_config);
+
+    if (ret != LIBUSB_SUCCESS) {
+        libusb_close(m_dev);
+        libusb_exit(m_ctx);
+
+        throw IOException("(%04X:%04X) Setting configuration failed: %s", m_vid, m_pid, libusb_error_name(ret));
+    }
+
```

Comment:
Why did you move setting the configuration after the kernel driver active check?

Reply:
Wouldn't work the other way around. Din had to do the same with for the ptapi lxio driver too.

Reply:
Also all the examples I see of people using libusb, the kernel check is before the set configuration. Still need to double check, but dont think there is any issue with mk6piuio doing it this way

Reply:
Tested with piuiomk6, and still works without issue.